src/ageml/ui.py

```diff
@@ -430,17 +696,13 @@ def features_vs_age(self, dfs: list, labels: list = None, significance: float =
                                                     feature_names[order_element], corr[order_element],
                                                     p_values[order_element]))
             # Append all the values
-            X_list.append(X)
-            y_list.append(y)
-            corr_list.append(corr)
-            order_list.append(order)
-            significance_list.append(significant)
+            X_list.append(X), y_list.append(y), corr_list.append(corr), order_list.append(order), significance_list.append(significant)
 
         # Use visualizer to show results
         self.visualizer.features_vs_age(X_list, y_list, corr_list, order_list,
-                                        significance_list, feature_names, labels, name)
+                                        significance_list, feature_names, tag, list(features_dict.keys()))
```

Comment:
Give a name to `list(features_dict.keys())`, for interpretability. e.g.: `feature_names`